components/virtual-node/virtual-node-rpcops-service-impl/src/main/kotlin/net/corda/virtualnode/rpcops/impl/validation/VirtualNodeValidationService.kt

```diff
@@ -0,0 +1,10 @@
+package net.corda.virtualnode.rpcops.impl.validation
+
+import net.corda.libs.packaging.core.CpiMetadata
+import net.corda.virtualnode.VirtualNodeInfo
+
+interface VirtualNodeValidationService {
```

Comment:
```suggestion
internal interface VirtualNodeValidationService {
```
to highlight the fact that it is not used outside of this module.